temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsImpl.java

```diff
@@ -45,15 +47,15 @@
 
   /** refers to the name of the gRPC header that contains the client library version */
   private static final Metadata.Key<String> LIBRARY_VERSION_HEADER_KEY =
-      Metadata.Key.of("temporal-client-version", Metadata.ASCII_STRING_MARSHALLER);
+      Metadata.Key.of("client-version", Metadata.ASCII_STRING_MARSHALLER);
 
-  /** refers to the name of the gRPC header that contains the client feature version */
-  private static final Metadata.Key<String> FEATURE_VERSION_HEADER_KEY =
-      Metadata.Key.of("temporal-client-feature-version", Metadata.ASCII_STRING_MARSHALLER);
+  /** refers to the name of the gRPC header that contains supported server versions */
+  private static final Metadata.Key<String> SUPPORTED_SERVER_VERSIONS_HEADER_KEY =
+      Metadata.Key.of("supported-server-versions", Metadata.ASCII_STRING_MARSHALLER);
 
   /** refers to the name of the gRPC header that contains the client SDK name */
-  private static final Metadata.Key<String> CLIENT_IMPL_HEADER_KEY =
-      Metadata.Key.of("temporal-client-name", Metadata.ASCII_STRING_MARSHALLER);
+  private static final Metadata.Key<String> CLIENT_NAME_HEADER_KEY =
+      Metadata.Key.of("client-name", Metadata.ASCII_STRING_MARSHALLER);
 
   private static final String CLIENT_IMPL_HEADER_VALUE = "temporal-java";
```

Comment:
rename to match key name to CLIENT_NAME_HEADER_VALUE.

Reply:
good eye